docs/docs/tutorial/tpch.md

```diff
@@ -122,7 +122,7 @@ CREATE TABLE IF NOT EXISTS orders (
   PRIMARY KEY (`o_orderkey`));
 
 -- lineitem
-CREATE TABLE IF NOT EXISTS lineitem`(
+CREATE TABLE IF NOT EXISTS lineitem (
```

Comment:
In MySQL, I didn't use explicit schema names and the quotes. But, to be consistent with the rest of the examples below, I think we should put them back. For example, lineitem -> `tpch1g`.`lineitem`. Then, the `use tpch1g` statement can be deleted above.